src/Container.js

```diff
@@ -0,0 +1,431 @@
+import { Bellhop } from 'bellhop-iframe';
+import { Features } from './Features';
+// @ts-ignore
+import { version } from '../package.json';
+
+//private/static variables
+let PLUGINS = [];
+const CLIENT = new Bellhop();
```

Comment:
We probably want a _per-instance_ `Bellhop` instance instead of a singleton here. This would allow for two Containers at the same time managing two separate iframes. Weird use case, but could totally occur...

Reply:
Moreover by locking the bellhop instance in scope, you prevent plugins from adding custom events into their games.